lua/gitlab/reviewer/init.lua

```diff
@@ -145,6 +145,7 @@ M.jump = function(file_name, line_number, new_buffer)
     line_number = number_of_lines
   end
   vim.api.nvim_win_set_cursor(0, { line_number, 0 })
+  u.open_fold_under_cursor()
```

Comment:
I'm also thinking about adding the following line, to redraw with the current line in the center of the window. It can make more context visible:
```lua
  vim.cmd("normal! zz")
```

Reply:
OK, I've just added that too, because I find myself always centering the window manually and not enjoying it at all :)